### PR TITLE
Copy updates and multiple style changes.

### PR DESCRIPTION
**Pull request by CristianPerez06**
## Summary

- Header
  - Updated links to have only: ``Home``, ``My Collection``, ``Mint a POAP`` and ``Community`` (removed ``Faqs``)
  - ``Community`` is a dropdown with the following options: ``Discord``, ``Telegram``, ``Twitter`` and ``Github``
- Body
  - Copy changes
  - Removed ``App Store`` and ``Play Store`` buttons
  - Added ``Drop a POAP`` button with new styles
  - Added ``See my collection`` button with new styles and ``Browse gallery`` link
  - Removed "How can I contact you or get involved" section
  - Removed "Why use POAP?" section
  - Updated fixed header
- Footer
  - Updated Ecosystem links to have only: ``poap.art``, ``poap.fun``, ``poap.gallery``, ``poap.vote``
  - Removed ``Reddit`` link from "Join our community" section.

> Link to [Figma](https://www.figma.com/file/MWxFRy43dEBObJa2npqLfH/NEW-LANDING-2022?node-id=9%3A4796)
> Additional comment: All of these changes were reviewed and approved by Hans
  
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Screenshots

> ### Before
> image
> image

> ### After
> image
> image
> image)
> image

## Testing

Go to the home page of POAP institutional web site and review each of the items included in the summary.

## Ticket

[PCD-715](https://poap-devs.atlassian.net/browse/PCD-715)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I didn't commit unnecessary logs

### CODE REVIEW

**Comment by sebasfavaron** [comment]
image
is this image above the ecosystem missing?

**Comment by sebasfavaron** [comment]
image

image
Header text should be Noway Round

File: public/index.html [hunk 1]
```diff
@@ -57,15 +57,43 @@
...
 
             <!-- Dev version -->
             <!-- <li><a href="faqs.html" class="anchor">FAQs</a></li> -->
-            
-            <li><a href='https://www.reddit.com/r/poap' class="anchor" target="_blank"><img src="assets/images/reddit2.svg" class="mlogo" alt="Reddit" /> Reddit</a></li>
-            <li><a href="https://github.com/poap-xyz" class="anchor"><img src="assets/images/logo-git2.svg" class="mlogo" alt="Github" />Github</a></li>
-            <li><a href="https://discord.gg/9s8U8Bn" class="anchor"><img src="assets/images/logo-discord2.svg" class="mlogo" alt="Discord" />Discord</a></li>
-            <li><a href="https://twitter.com/poapxyz" class="anchor"><img src="assets/images/logo-twitter2.svg" class="mlogo" alt="twitter" />@poapxyz</a></li>
+            <li><a href='https://app.poap.xyz/scan/' class="anchor">My collection</a></li>
+            <li><a href='https://app.poap.xyz/admin/' class="anchor">Mint a POAP</a></li>
```
Comment (sebasfavaron) [inline public/index.html:65]: Links that point outside the website should open in a new tab

**Comment by CristianPerez06** [comment]
> image is this image above the ecosystem missing?

Hi @sebasfavaron remember this ticket is about updating the text and adding some new buttons (which obviously require new styles) but it's not about re-styling the website. Some things may differ from what we see in Figma but we should only focus on what was asked to do in the ticket (also explained in the summary of this PR to make things easier).

**Comment by CristianPerez06** [comment]
> image
> 
> image Header text should be Noway Round

Same [here](https://github.com/poap-xyz/poap-website/pull/17#issuecomment-980643134)

File: public/index.html (hunk 1 above)
Comment (CristianPerez06) [reply public/index.html:65]: Good catch! I'll make that change

**Review by sebasfavaron — APPROVED** [review]

**Pull request closed without merging**